src/core/proof.ml

```diff
@@ -60,45 +73,132 @@ type proof_state =
 (** Short synonym for qualified use. *)
 type t = proof_state
 
-(** [init builtins name a] returns an initial proof state for a theorem  named
+(** [init name a] returns an initial proof state for a theorem  named
     [name], which statement is represented by the type [a]. Builtin symbols of
-    [builtins] may be used by tactics, and have been declared. *)
+    [builtins] may be used by tactics, and have been declared.
```

Comment:
There is no builtins argument anymore. This should be removed.

Reply:
fixed in ed44999